debian/control

```diff
@@ -0,0 +1,54 @@
+Source: folly
+Maintainer: Ricky Ramirez <rram@reddit.com>
+Section: libs
+Priority: extra
+Standards-Version: 3.9.5
+Build-Depends: debhelper (>= 9),
+ autoconf-archive,
+ automake,
+ dh-autoreconf,
+ gcc-4.8,
+ g++-4.8,
+ libboost1.54-dev,
+ libboost-context1.54-dev,
+ libboost-filesystem1.54-dev,
+ libboost-program-options1.54-dev,
+ libboost-python1.54-dev,
+ libboost-regex1.54-dev,
+ libboost-system1.54-dev,
+ libboost-thread1.54-dev,
+ libdouble-conversion-dev,
+ libevent-dev,
+ libgoogle-glog-dev,
+ libgflags-dev,
+ libgtest-dev,
+ libsnappy-dev,
+ libssl-dev,
+ libtool,
+ pkg-config,
+ python-dev
+
+Package: libfolly55
+Section: libs
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: Facebook Open-source LibrarY
```

Comment:
Was intentional, but looks like they got rid of it: https://github.com/facebook/folly/commit/b85957465746e9275528ceeb4d6646094a586874